travisci/pylint_harness.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2019] EMBL-European Bioinformatics Institute
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#      http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+tmpfile=$(mktemp)
+
+pylint --rcfile pylintrc scripts/ | grep -v "\-\-\-\-\-\-\-\-\-" | grep -v "Your code has been rated" | grep -v "\n\n" | sed '/^$/d' > "$tmpfile"
```

Comment:
Should we include also the future `src/python/`? Or will it fail/complain because that path doesn't exist at the moment?

Reply:
It fails if asked to run on a file that doesn't exist. It should be added in #157